.sauce/config.yml

```diff
@@ -2,28 +2,27 @@ version: 2.1
 apiVersion: v1alpha
 kind: espresso
 sauce:
-  region: us-west-1
+  region: eu-central-1
```

Comment:
Any specific reason for shifting it to eu-central?

Reply:
I moved to eu-central as I am UK based.  Performance is a lot better for me for obvious reasons.   I'm ok to revert for this PR...